api/events/src/test/java/io/opentelemetry/api/events/DefaultEventEmitterTest.java

```diff
@@ -21,7 +21,9 @@ void emit() {
     assertThatCode(
             () ->
                 DefaultEventEmitter.getInstance()
-                    .emit("event-name", Attributes.builder().put("key1", "value1").build()))
+                    .emit(
+                        "otel.scope.event-name",
```

Comment:
`otel.scope.` made me think it was instrumentation scope for a minute (unless "scope" is what we're calling this in events?)

```suggestion
                        "event-domain.event-name",
```

Reply:
yeah good call, that tried to be helpful and made it worse. :)